git_machete/annotation.py

```diff
@@ -37,11 +39,17 @@ def sub_pattern(text: str) -> str:
             self.__slide_out_text = 'slide-out=no'
             self.__annotation_without_qualifiers = re.sub(sub_pattern('slide-out'), ' ', self.__annotation_without_qualifiers)
 
+        merge_match = re.match(match_pattern("update", value="merge"), annotation)
+        if merge_match:
+            self.merge = True
+            self.__merge_text = "update=merge"
+            self.__annotation_without_qualifiers = re.sub(sub_pattern('update', value='merge'), ' ', self.__annotation_without_qualifiers)
+
     def get_annotation_text_without_qualifiers(self) -> str:
         return self.__annotation_without_qualifiers.strip()
 
     def get_qualifiers_text(self) -> str:
-        return f'{self.__rebase_text} {self.__push_text} {self.__slide_out_text}'.replace('  ', ' ').strip()
+        return f'{self.__rebase_text} {self.__push_text} {self.__slide_out_text} {self.__merge_text}'.replace('  ', ' ').strip()
```

Comment:
Note that now you can end up with triple space in the output, so maybe use `re.sub(' *', ' ', ...)`